openpace: init at 1.1.3

vaavaav opened the pull request:
- Add OpenPACE, a cryptographic library for EAC version 2.
- Contrary to the official documentation, the `--enable-openssl-install` flag is not used, as it does not follow Nix standards and was removed in later commits.
- Uses OpenSSL 3.x for compatibility.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ethancedwards8 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 386287`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>openpace</li>
  </ul>
</details>

ethancedwards8 commented on pkgs/by-name/op/openpace/package.nix:
```diff
@@ -0,0 +1,42 @@
+  preConfigure = ''
+    autoreconf --verbose --install
+  '';
+
```
```suggestion
passthru.updateScript = nix-update-script { };
```

ethancedwards8 commented:
Squash the commits

ethancedwards8 commented:
Also split the maintainer and package files into different commits

Commit:
maintainers: add vaavaav

Commit:
openpace: init at 1.1.3

- Add OpenPACE, a cryptographic library for EAC version 2.
- Contrary to the official documentation, the --enable-openssl-install flag is not used, as it does not follow Nix standards and was removed in later commits.
- Uses OpenSSL 3.x for compatibility.

FliegendeWurst reviewed (commented):
Is the presence of `$out/bin/example` intentional?
```
results/openpace-x86_64-linux
├── bin
│   ├── cvc-create
│   ├── cvc-print
│   ├── eactest
│   └── example
```

vaavaav commented:
No... Glad you caught that, I actually didn't notice.

FliegendeWurst commented:
I pushed to

- correct the wording of the maintainer commit
- add meta.broken = true for cross compilation due to use of help2man
